bot/plugins/schedule.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from bot.config import Config
+from bot.data import command
+from bot.data import format_msg
+from bot.message import Message
+
+
+@command('!schedule')
+async def cmd_schedule(config: Config, msg: Message) -> str:
+    return format_msg(
+        msg,
```

Comment:
bot/plugins/simple.py is more intended for these one-off ones

Reply:
Thank you, I missed that one. This was a copy of the `Kevin.py` file I changed. I'll add it there and update the PR